model/span.go

```diff
@@ -254,6 +255,7 @@ func (e *Span) fields(ctx context.Context, cfg *transform.Config) common.MapStr
 
 	fields.maybeSetMapStr("db", e.DB.fields())
 	fields.maybeSetMapStr("http", e.HTTP.fields())
+	fields.delete("http.response.headers")
```

Comment:
> I think this is backwards? http.response.headers doesn't exist in ECS, so we should only set it as span.http.response.headers for now

OK, I thought we wanted to move everything up because after 8.0 is going to be weird that some http fields are top level, and others are inside spans. Like, if they end up in ECS we would have them already at the top level, otherwise they would not hurt.